README.md

```diff
@@ -127,6 +127,26 @@ Custom registries are not currently supported
 ([#160](https://github.com/obi1kenobi/cargo-semver-checks/issues/160)), so crates published on
 registries other than crates.io should use one of the other approaches of generating the baseline.
 
+#### Git repository detection
+
+When looking up a git revision with `--baseline-rev`,
+`cargo-semver-checks` will walk up the current directory until it finds a `.git/` directory to resolve the revision and extract the corresponding worktree.
+You can use the following environment variables to influence the git repository detection
+
+- [`GIT_CEILING_DIRECTORIES`] to stop automatic git repository discovery when reaching certain directories.
```

Comment:
The "certain directories" phrase is a bit confusing. Could you offer a bit of intuition to the reader as to when they might use this?

Also, what's the default value / behavior here?